### DISCUSSION TITLE
pagination arrows

### DISCUSSION THREAD

**Discussion by albukzek** [quasarframework/quasar#14252] [General - Components / Directives / etc]
Hi all. Please tell me how to add your own images of arrows in pagination.

I tried like this but it doesn't work

`<q-pagination
      v-model="current"
      icon-first="img:@\img\arrowl.svg"
      icon-last="img:@\img\arrowr.svg"
    />`

**Comment by MilosPaunovic** [comment]
I think you're looking for `icon-prev-page` & `icon-next-page` props, and you should use relative paths, e.g. `img:../path/to/image.svg`
